### PR TITLE
logo sizes has been increased

### PR DESCRIPTION
**Pull request by faisalamin001**
## Related Issue

- Information about the related issue

Closes: #190

### Describe the changes you've made

I increased the logo sizes both in the Nav bar and footer.

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update.
- [ ] This change requires a documentation update

## Checklist

- [ ] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented on my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)

 Original           | Updated
 :--------------------: |:--------------------:
 original screenshot | updated screenshot |

## Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/WebXDAO/.github/blob/main/CODE_OF_CONDUCT.md)

<a href="https://gitpod.io/#https://github.com/WebXDAO/WebXDAO.github.io/pull/191"></a>

### CODE REVIEW

**Comment by Panquesito7** [comment]
It still seems to be wrong/very stretched.

image

File: src/components/Global/Footer.js [hunk 1]
```diff
@@ -38,17 +38,17 @@ const navigation = [
...
+    <footer className='bg-transparent'>
+      <div className='max-w-screen-xl px-4 pt-16 pb-6 mx-auto sm:px-6 lg:px-8 lg:pt-24'>
+        <div className='grid grid-cols-1 gap-8 lg:grid-cols-3'>
           <div>
-            <div className="flex justify-center text-teal-600 sm:justify-start">
-              <a href="#">
-                <span className="sr-only">WebX DAO</span>
+            <div className='flex justify-center text-teal-600 sm:justify-start'>
+              <a href='#' className=''>
+                <p className='sr-only '>WebX DAO</p>
```
Comment (mkubdev) [inline src/components/Global/Footer.js:47]:
```suggestion
                <span className="sr-only">WebX DAO</span>
```
This shouldn't be a paragraph. It's a invisible span used for A11y.

File: src/components/Global/Footer.js [hunk 2]
```diff
@@ -38,17 +38,17 @@ const navigation = [
...
-        <div className="grid grid-cols-1 gap-8 lg:grid-cols-3">
+    <footer className='bg-transparent'>
+      <div className='max-w-screen-xl px-4 pt-16 pb-6 mx-auto sm:px-6 lg:px-8 lg:pt-24'>
+        <div className='grid grid-cols-1 gap-8 lg:grid-cols-3'>
           <div>
-            <div className="flex justify-center text-teal-600 sm:justify-start">
-              <a href="#">
-                <span className="sr-only">WebX DAO</span>
+            <div className='flex justify-center text-teal-600 sm:justify-start'>
+              <a href='#' className=''>
```
Comment (mkubdev) [inline src/components/Global/Footer.js:46]:
```suggestion
              <a href='#'>
```
No need of empty `className`

File: src/components/Global/Footer.js [hunk 3]
```diff
@@ -38,17 +38,17 @@ const navigation = [
...
+                <p className='sr-only '>WebX DAO</p>
                 <img
-                  className="w-auto h-8 sm:h-10"
-                  src="/images/logo/white_logo.png"
-                  alt=""
+                  className='h-72  w-20 sm:h-10'
```
Comment (mkubdev) [inline src/components/Global/Footer.js:49]: Hi, this can't be scalled. You need to use `w-auto` and update the `height` ☺️

**Review by mkubdev — CHANGES_REQUESTED** [review]
Hi, thanks for your contribution. As @Panquesito7 mentionned, logo is still stretched and the codestyle isn't good. You replaced every `"` by `'`.

Please update ✨

**Comment by faisalamin001** [comment]
Fixed

**Comment by faisalamin001** [comment]
Screenshot 2022-09-21 at 10 14 17 PM

**Pull request closed without merging**